tethne/readers/zotero.py

```diff
@@ -80,11 +80,13 @@ def handle_link(self, value):
                 return str(o).replace('file://', '')                 
 
     def handle_date(self, value):
-        try:
-            return iso8601.parse_date(str(value)).year    
-        except iso8601.ParseError:
-            return datetime.strptime(str(value), "%m/%d/%Y").date().year
-        
+    		for datefmt in ("%B %d, %Y", "%Y-%m", "%Y-%m-%d", "%m/%d/%Y"):
+		        try:
+		            print datetime.strptime(str(value), datefmt).date().year
+		        except iso8601.ParseError:
```

Comment:
How would an iso8601.ParseError occur if you don't use any methods from the iso8601 module?